tutor/src/components/book-page.js

```diff
@@ -245,8 +245,8 @@ class BookPage extends React.Component {
       }
 
       // remove all non-lists children to prevent extra text in preamble
-      if (typeof abstractChild.remove === 'function') {
-        abstractChild.remove();
+      if (true !== invoke(abstractChild, 'matches', '[data-type="title"]')) {
```

Comment:
Won't matches return an array of matches or something instead of `true`?

Reply:
nah, that calls the `matches` method on an element, it returns a bool: https://developer.mozilla.org/en/docs/Web/API/Element/matches